jest.config.js

```diff
@@ -0,0 +1,35 @@
+module.exports = {
```

Comment:
will clear which are not required
copied from https://medium.freecodecamp.org/how-to-set-up-jest-enzyme-like-a-boss-8455a2bc6d56